code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -1047,6 +1051,37 @@
 	playsound(hugger, 'sound/effects/xeno_newlarva.ogg', 25, TRUE)
 	hugger.generate_name()
 
+/datum/hive_status/proc/update_lesser_drone_limit()
+	lesser_drone_limit = Ceiling(totalXenos.len / 3)
```

Comment:
The issue here is that totalXenos list includes all xenos, including facehuggers and lesser drones. So every time you join as lesser drones, you essentially increase the limit for both facehuggers and lesser drones. 

Also, same issue with facehuggers, they increase the limit for both parties as well.

Reply:
This is a consideration and one I am happy to leave as is.

Reply:
Just to illustrate the issue, during chance claim round at some point xenos had 14 actual xenos, 10/10 lesser drones and 5/(?, likely also 10) facehuggers. If all facehugger slots were filled, limits would be even higher. Just pointing out.

Reply:
Its a bit messy but there is `/datum/hive_status/proc/get_xeno_counts()` that is excluding all that have counts_for_slots false and those amounts can be tallied up like they are for the hive status UI `/datum/hive_status_ui/proc/update_xeno_counts(send_update = TRUE)` so if you do decide to go that way, can likely replicate that code or adapt it in a way that its more cached so hivestatus tgui and this can share the same data.